### ISSUE TITLE
Position option

### ISSUE THREAD

**Issue by ivoreissilva** [jboesch/Gritter#12]
Hello,

I was trying to specify the position option when adding a new notification but by looking at the source code I saw that you're doing this (add method - around line 107):

position = $.gritter.options.position

I just want to know if there is any restriction that is preventing you to do something like:

position = params.position||$.gritter.options.position

Allowing different notices to be displayed in different positions.

Thanks in advance!

**Comment by jboesch** [comment]
That's definitely a possibility. I just can't understand why a person would want to change where the gritter message shows from notice to notice. For consistency, it's probably better that the user gets used to it showing in one place. Maybe I can be convinced otherwise.

**Comment by ivoreissilva** [comment]
Hi, Jordan!

I understand it's not the usual way to use gritter (growl) but I can give you a scenario. This, of course, can be argued against since it has it's pros and cons.

Imagine you have a page divided in two areas (50 / 50) and you're displaying different information in each side.
You can affect this information, with forms and buttons, using AJAX mechanisms.

While you're affecting this information I believe it would be nice to the user to see related messages on the corresponding side (top-left / top-right).

Clearly I could use my own messages display or even use other plug-ins (dialogs, overlays...) but the gritter plug-in does the job quite nicely and has an interesting look.

Once again, this a particular scenario and you might consider that defeats the original growl implementation and I accept that.

Best regards!

**Comment by jboesch** [comment]
I looked at the code. Unfortunately this won't work as an easy fix. As soon as you drop in params.position for a gritter element, all the gritter notices become wrapped in that wrapper. As in, it doesn't create a new wrapper for some to be positioned top-right or some to be positioned top-left. 

This might be something to look at down the road. But I've decided to not go ahead with it at this moment.
Thanks for your feedback!
